src/os/bluestore/Allocator.h

```diff
@@ -53,7 +53,8 @@ class Allocator {
   void release(const PExtentVector& release_set);
 
   virtual void dump() = 0;
-  virtual void dump(std::function<void(uint64_t offset, uint64_t length)> notify) = 0;
+  virtual void foreach(
```

Comment:
@ifed01 https://jenkins.ceph.com/job/ceph-pull-requests/93435/ looks related

Reply:
yep, sorry! Fixing...

Reply:
no worries, looks good now! I don't think any further teuthology testing is required, would you agree?